### ISSUE TITLE
[Bitvavo] errorCode 304

### ISSUE THREAD

**Issue by trip3131** [ccxt/ccxt#8890]
Hi, when trying to connect to Bitvavo I get a 403 and 304 error. I'm sure that my key and secret are correct. I am new to CCXT. Previously I used the Python wrapper provided by Bitvavo without much problem. Maybe I'm making a newbie mistake. Any suggestions on what goes wrong?

####PYTHON CODE####
import ccxt

exchange = ccxt.bitvavo({
    'enableRateLimit': True,
    'apiKey': 'myKey',
    'secret': 'mySecret',
    })

print(exchange.load_markets()) # This works
print(exchange.fetch_balance()) # This does not work

####ERROR#### (I leave out the complete traceback):

HTTPError: 403 Client Error: Forbidden for url: https://api.bitvavo.com/v2/balance

During handling of the above exception, another exception occurred:

AuthenticationError: bitvavo {"errorCode":304,"error":"Request was not received within acceptable window (default 30s, or custom with Bitvavo-Access-Window header) of Bitvavo-Access-Timestamp header (or timestamp parameter for websocket)."}

**Comment by kroitor** [comment]
Hi! Have you tried creating a fresh new keypair?

**Comment by trip3131** [comment]
> 
> 
> Hi! Have you tried creating a fresh new keypair? If that does not help, you might have to configure syncing your system clock every 10 minutes.

I created a new keypair for CCXT indeed (I tried multiple in fact). 

My windows clock was synced about a month ago, so I hoped that resyncing would solve the issue.  However, it did not had an immediate effect.

Then, I restarted my Python kernel. I was still using the same IPython session. Now the error message is gone. Thank you for the suggestion about the syncing, so it was that indeed.

I have to admit that I am still a bit confused about the system time, the time Python uses, and the requirements of Bitvavo. Hopefully, I don't encounter it again. 

Thank you.
